src/_components/TableHeaderComponent.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
As the `TableHeaderComponent` will be re-used in other components, please move it to the `common` folder which is on this path `src/components/common`

Reply:
Done that. Thank you.